build.gradle

```diff
@@ -142,11 +117,26 @@ dependencies {
     testImplementation 'junit:junit:4.13.2'
 }
 
-description = 'Apache Camel IDE :: IDEA Plugin'
+intellijPlatform {
+    pluginVerification {
+        ides {
+            recommended()
```

Comment:
IMO, I'm not really a fan of losing the list of versions officially supported (reason why I didn't do it during the migration), it is already complex to support the 4 latest versions if we no longer keep control of the list, it will be even worse. 

@davsclaus WDYT?